index.html

```diff
@@ -18,14 +18,132 @@
 
   <link rel="stylesheet" href="css/normalize.css">
   <link rel="stylesheet" href="css/main.css">
-
+<!--  <link rel="stylesheet" href="http://use.fontawesome.com/releases/v5.2.0/css/all.css">-->
   <meta name="theme-color" content="#fafafa">
-</head>
+  <script src="https://kit.fontawesome.com/b32a079d69.js" crossorigin="anonymous"></script>
 
+</head>
 <body>
+<!-- Add your site or application content here -->
+
+<main-header></main-header>
+
+<div class="black"></div>
+
+<section class = "welcome">
+  <h1>Welcome</h1>
+  <h2>Car repair services</h2>
+  <p>Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim, quis nostrud nisi ut aliquip ex ea.</p>
+  <button>More info</button>
+  <button>Contact us</button>
+  <img class = "car" src="img/car.png">
+</section>
+
+<section class = "features clearfix">
+    <div class="feature">
+      <h4>Auto diagnose</h4>
+      <img src="img/thr.png">
+      <p class="descrip">Lorem ipsum dolor sit quis aliquip ex ea commodo consequat.</p>
+    </div>
+    <div class="feature">
+      <h4>Auto diagnose</h4>
+      <img src="img/thr1.png">
+      <p class="descrip">Lorem ipsum doitation nisi ut aliquip ex ea commodo consequat.</p>
+    </div>
+    <div class="feature">
+      <h4>Auto diagnose</h4>
+      <img src="img/thr2.png">
+      <p class="descrip">Lorem ipsum dolor sit amet, c magna aliqua. Ut enim ad minim veniam.</p>
+    </div>
+    <button class = "arrow"><p>></p></button>
+
+
+  </section>
+
+<main>
+  <section class = "about">
+    <h2>About Our Car Services</h2>
+    <div>
+      <img src="img/about.png">
+      <h4>Dolor sit amet, consectetus adipiscing elit, </h4>
+      <p>lorem ipsum dolor sit amet, dipiscin sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, sed do eiusmod tempor incididunt ut labore  quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum</p>
+      <button>Read More</button>
+
+    </div>
+  </section>
+  <section class = "we-do">
+    <h2>What We Do</h2>
+    <p>It is a long established fact that a reader will be distracted by the</p>
+    <div class = "images">
+      <img src="img/img1.png">
+      <img src="img/img2.png">
+      <img src="img/img3.png">
+      <img src="img/img4.png">
+    </div>
+    <button class= "see-more">See more</button>
+  </section>
+  <section class= "testimonial">
+    <h2>Testimonial</h2>
+    <p>It is a long established fact that a reader will be distracted by the</p>
+    <div class="jckolo">
```

Comment:
jckolo? :) Klasy powinny dosyć dokładnie opisywać czym jest dany element tak, żeby w razie czego łatwo można było to wyszukać :)